ui/components/requirements/tabs.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+function Tab({ active, link, tabName }) {
+  if (active) {
+    return <li className="inline-block mr4 bold">{ tabName }</li>;
+  }
+  return (
+    <li className="inline-block mr4">
+      <Link to={link} >{ tabName }</Link>
+    </li>
+  );
+}
+Tab.defaultProps = {
+  active: false,
+  link: { pathname: '', query: {} },
+  tabName: '',
+};
+Tab.propTypes = {
+  active: React.PropTypes.bool,
+  link: React.PropTypes.shape({
+    pathname: React.PropTypes.string,
+    query: React.PropTypes.shape({}),
+  }),
+  tabName: React.PropTypes.string,
+};
+
+export default function Tabs({ router }) {
```

Comment:
From my understanding, functions are preferred to class based components whenever we aren't tracking any state (which is the bulk of our components). AirBnB's lint configuration will actually yell at you if you don't follow this strategy.